docs/K8SCollectors.md

```diff
@@ -0,0 +1,179 @@
+# **K8S data collectors**
+The Splunk Operator for K8S deploys Splunk Enterprise custom resources across a single/multiple namespace/s. The helper scripts `k8s-splunk-collector.sh` and `k8s-systeminfo-collector.sh` in the [tools directory](https://github.com/splunk/splunk-operator/tree/master/tools/k8s_collectors), collect data from a K8S cluster which runs the Splunk Operator for K8S.
```

Comment:
Changing "a single/multiple namespace/s" to "a single namespace or multiple namespaces" would be easier to read and not much longer.